client/src/components/Review.jsx

```diff
@@ -1,18 +1,44 @@
-import React from 'react';
+import React, { useState } from 'react';
+
+// this component will render individual reviews
+export default ({ review }) => {
+  // convert ISO date-time to string and add to state
+  let rawDate = '';
+  if (review.date) {
+    rawDate = new Date (review.date).toString().split(' ').slice(1, 4);
+    rawDate[1] = rawDate[1] + ', ';
+    rawDate = rawDate.join(' ');
+  }
+
+  // check content needs to be truncated
+  const limitContent = review.content.length > 40 ? true : false;
+
+  //add to state
+  const [date, setDate] = useState(rawDate);
+  const [readLimit, setLimit] = useState(limitContent);
+
+  // function to remove readLimit to see all content
+  const handleReadMore = () => {
+    setLimit(false);
+  };
+
+  const ReadMore = () => <a className="blue-links" onClick={handleReadMore}>&nbsp; Read more</a>;
 
-export default () => {
   return (
     <div className="reviews-section-card">
-      <p className="reviews-section-author">Real Person</p>
-      <p className="reviews-section-date">10 November 1775</p>
+      <p className="reviews-section-author">{ review.author }</p>
+      <p className="reviews-section-date">{ date || '' } </p>
       <div className="reviews-section-stars">
         <span className="fas fa-star fa-xs star-small"></span><span className="fas fa-star fa-xs star-small"></span><span className="fas fa-star fa-xs star-small"></span>
       </div>
       <div className="reviews-section-content">
-          Lorem ipsum, dolor sit amet consectetur adipisicing elit. Voluptates, dolorum. Molestiae dolore repellat, laboriosam quia rerum labore perspiciatis architecto tempore expedita dolor iste quis eaque at earum dolorem minus temporibus eius ducimus deserunt inventore consequatur! 
-          <br/>
-          <a href="">Read more</a>
+        <p>
+          {/* if content is long, truncate and display read more button */}
+          { readLimit ? review.content.slice(0, 40).concat(' ...') : review.content }
+          { readLimit ?  <ReadMore /> : '' }
```

Comment:
Does a template literal work here? Idk if this is valid, but if it is it makes the code easier to understand
```
{ readLImit ? `${review.content.slice(0, 40)} ... ${<ReadMore />}` : review.content }
```

Reply:
Yeah it definitely works and looks alot better. Will update